src/components/TasksTable/index.jsx

```diff
@@ -30,6 +30,20 @@ const sorted = pipe(
   },
   chip: {
     marginRight: 1,
+    '&:nth-of-type(4n+0)': {
+      backgroundColor: theme.palette.primary.dark,
+      color: 'white',
+    },
+    '&:nth-of-type(4n+1)': {
+      backgroundColor: theme.palette.secondary.dark,
+      color: 'white',
+    },
+    '&:nth-of-type(4n+2)': {
+      backgroundColor: theme.palette.secondary.main,
+    },
+    '&:nth-of-type(4n+3)': {
+      backgroundColor: theme.palette.secondary.light,
+    },
```

Comment:
Having multiple colors without any meaning could be confusing. Let's go with `backgroundColor: SECONDARY.DARK`.

Let's define this style in theme.jsx:

```jsx
MuiChip: {
  root: {
    height: 24,
    marginRight: 1,
    backgroundColor: SECONDARY.DARK,
    color: 'white',
  },
},
```

You can now delete the `chip` class.

Reply:
This still needs to be removed.